src/moscot/base/problems/_utils.py

```diff
@@ -721,5 +721,5 @@ def _get_n_cores(n_cores: Optional[int], n_jobs: Optional[int]) -> int:
     return n_cores
 
 
-def _compute_conditional_entropy(p_xy: ArrayLike) -> ArrayLike:
+def _compute_conditional_entropy(p_xy: ArrayLike, c: float = 0.0) -> ArrayLike:
```

Comment:
We can remove this function then , right?

Reply:
yes done